homeassistant/components/ambient_network/strings.json

```diff
@@ -0,0 +1,92 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Region Selection",
+        "description": "Choose the region you want to survey in order to locate Ambient personal weather stations.",
+        "data": {
+          "latitude": "[%key:common::config_flow::data::latitude%]",
+          "longitude": "[%key:common::config_flow::data::longitude%]"
+        }
```

Comment:
Don't we use the location selector? -> does these strings even do anything?

Reply:
Removed.